.github/workflows/ci.yml

```diff
@@ -139,6 +139,49 @@ jobs:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
 
+  docker:
+    needs:
+      - lint
+      - test-unit
+      - test-integration
```

Comment:
Probably also want to add `test-integration-windows`?

Reply:
Ok, will do